src/bundle/Resources/views/Limitation/udw_limitation_value_tag.html.twig

```diff
@@ -0,0 +1,17 @@
+<li class="mt-2">
+    <div class="ez-tag" data-location-id="{{ location_id }}">
+        <div class="ez-tag__content" {% if is_loading_state %} hidden {% endif %}>
+            {{ content_breadcrumbs }}
+        </div>
+        <div class="ez-tag__spinner" {% if not is_loading_state %} hidden {% endif %}>
+            <svg class="ez-icon ez-icon--medium ez-icon--dark ez-spin">
+                <use xlink:href="/bundles/ezplatformadminui/img/ez-icons.svg#spinner"></use>
```

Comment:
```suggestion
                <use xlink:href="{{ asset('/bundles/ezplatformadminui/img/ez-icons.svg') }}#spinner"></use>
```

otherwise whole `assets` configuration (https://symfony.com/doc/3.4/reference/configuration/framework.html#assets) will be ignored. 

This might be an issue for projects which use CDN or versioning for static resources. For the following example configuration (/app/config/config.yml): 

```yml 
framework:
    assets:
        version: 'v2'
        base_urls:
            - 'https://cdn1.mysite.com'
```

`href` attribute value should be `https://cdn1.mysite.com/bundles/ezplatformadminui/img/ez-icons.svg?v2#spinner` rather then `/bundles/ezplatformadminui/img/ez-icons.svg#spinner`